### PR TITLE
3738 media migration

### PR DESCRIPTION
**Pull request by elliottjf**

### CODE REVIEW

File: db/migrate/20160201223840_add_uploader_to_media.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUploaderToMedia < ActiveRecord::Migration
+  def change
+    add_reference :media, :uploader, references: :people
```
Comment (linesarefuzzy) [inline db/migrate/20160201223840_add_uploader_to_media.rb]: Add foreign key?

**Comment by linesarefuzzy** [comment]
Looks good. I assume you've tested it with some actual media and it works? Go ahead and resolve conflicts and merge.

File: db/migrate/20160201223840_add_uploader_to_media.rb (hunk 1 above)
Comment (elliottjf) [inline db/migrate/20160201223840_add_uploader_to_media.rb]: added!

**Comment by elliottjf** [comment]
yep. tested with actual media.  note that the current media uploader implementation is hardwired to 'uploads'.  on my system, i just mapped around that with symlinks, but we'll likely want to revisit before proceeding with a production configuration.  (i didn't want to mess with mel's original media work as part of this branch)

**Pull request merged**